askama_mendes/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "askama_mendes"
+version = "0.1.0"
+authors = ["Dirkjan Ochtman <dirkjan@ochtman.nl>"]
```

Comment:
In #562 you removed the author fields.

Reply:
Yeah, but this PR is independent of that one. Depending on which one lands first, I'll fix up the other one.

Reply:
Hehe, now this is the second PR to land. :) Approved after you cut this one line.